src/paperless/utils.py

```diff
@@ -0,0 +1,9 @@
+import unicodedata
+
+
+def make_searchable(content):
+    return (
+        unicodedata.normalize("NFKD", content.casefold())
+        .encode("ASCII", "ignore")
```

Comment:
Hm, I'm not sure if this is the right thing to do. In light of [your change only searching the new searchable fields](https://github.com/the-paperless-project/paperless/pull/545/files#diff-8515db01a71e5ae1a53884301d7dc1f7L176-R182), this straight up breaks search for languages that share no characters with the ASCII codepage, right? Judging from the [official Python docs](https://docs.python.org/3/howto/unicode.html#converting-to-bytes), `"ignore"` would simply drop non-ASCII characters, resulting in a potentially empty search-term.

As far as I can tell not ignoring these characters here would break the entire feature, your `Zürich Weiß` example would, after case-folding first, produce `zu\xcc\x88rich weiss`, which obviously cannot be searched for with `zurich weiss`.

Maybe the simplest fix is to search both the new ASCII-fied fields _and_ the regular fields?